boards/arm/mps3_an547/Kconfig.defconfig

```diff
@@ -6,6 +6,14 @@ if BOARD_MPS3_AN547
 config BOARD
 	default "mps3_an547"
 
+# MPU-based null-pointer dereferencing detection cannot
+# be applied as the (0x0 - 0x400) is unmapped but QEMU
+# will still permit bus access.
+choice NULL_POINTER_EXCEPTION_DETECTION
```

Comment:
Is this a QEMU bug?

Reply:
This was actually copied off the `mps2_an521` board:

https://github.com/zephyrproject-rtos/zephyr/blob/f3435b75efbfd7614ff873f8bfa90a031d6d27e8/boards/arm/mps2_an521/Kconfig.defconfig#L6-L12

The Cortex-M null pointer detection (MPU mode) relies on the MPU to generate faults when the 0x0-0x400 "page" is accessed, and it does not do that in case of QEMU according to the comment (I haven't tested it on FVP).

https://github.com/zephyrproject-rtos/zephyr/blob/b3b36f69a6444baac1cad1a67590e5fd7af70cf4/arch/arm/core/aarch32/mpu/arm_mpu.c#L356-L373